tests/unit/test_config.py

```
@@ -251,3 +251,106 @@ def test_api_finding_search_endpoint(self):
+		self.assertIn('mongodb://user:pass@host', yaml_str)
+
+	def test_get_print_false_returns_secret_str(self):
+		from pydantic import SecretStr
+		from secator.config import Config
+		config = Config.parse({'addons': {'mongodb': {'url': 'mongodb://user:pass@host'}}})
+		value = config.get('addons.mongodb.url', print=False)
+		self.assertIsInstance(value, SecretStr)
+		self.assertEqual(value.get_secret_value(), 'mongodb://user:pass@host')
+
+	def test_save_writes_real_value(self):
+		import yaml
```

Flake8 (7.3.0)

[error] 'yaml' imported but unused

(F401)